modules/ebi-metagenomics/owltools/meta.yml

```diff
@@ -0,0 +1,54 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "owltools"
+description: OWLTools is convenience java API on top of the OWL API, used here for mapping GO terms to GO-slims
+keywords:
+  - owl
+  - ontology
+  - gene ontology
+  - go
+tools:
+  - "owltools":
+      description: "OWLTools is convenience java API on top of the OWL API, used here for mapping GO terms to GO-slims"
+      homepage: "https://github.com/owlcollab/owltools"
+      documentation: "https://github.com/owlcollab/owltools"
+      tool_dev_url: "https://github.com/owlcollab/owltools"
+      licence: ["BSD-3-Clause"]
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - input_gaf:
+        type: file
+        description: Input GAF file
+        pattern: "*.gaf"
```

Comment:
This GAF files it's generated by one the toolkit scripts, right? I need to add step in pipeline to do so.

Reply:
Ignore this.. :dumb: from me. This is reason for this module to exist.